incoming/package_events.md

```diff
@@ -134,28 +134,28 @@ The identifier must be part of the event data. Otherwise, only changed fields ca
 
 `object` with following properties:
 
-| Property            | Type    | Required     | Description |
-| ------------------- | ------- | ------------ | ------- |
-| `id`                | string  | **Required** | GUID (must be generated and can't be changed)
-| `availableFrom`     | string  | **Optional** | yyyy-MM-dd
-| `availableTo`       | string  | **Optional** | yyyy-MM-dd
-| `calculatedCount`   | integer | **Optional** | 
-| `consistsOfCount`   | integer | **Optional** | 
-| `consistsOfUnit`    | string  | **Optional** | From reference data, eg. "STK"
-| `dPakLayerCount`    | integer | **Optional** |
-| `deliverable`       | boolean | **Optional** |
-| `gtin`              | string  | **Optional** |
-| `height`            | decimal | **Optional** | millimeters
-| `length`            | decimal | **Optional** | millimeters
-| `maxStackingWeight` | decimal | **Optional** | 
-| `minOrderQuantity`  | integer | **Optional** |
-| `packageNumber`     | integer | **Optional** |
-| `stocked`           | boolean | **Optional** |
-| `type`              | string  | **Optional** | One of "F-PAK", "D-PAK", "T-PAK", "M-PAK"
-| `unit`              | string  | **Optional** | From reference data, eg. "STK"
-| `volume`            | decimal | **Optional** | dm3
-| `weight`            | decimal  | **Optional** | kg
-| `width`             | decimal | **Optional** | millimeters
+| Property            | Type    | Required     | Description 					| Riversand Comment                                 	|
+| ------------------- | ------- | ------------ | ---------------------------------------------- | ----------------------------------------------------- |
+| `id`                | string  | **Required** | GUID (must be generated and can't be changed)	| This will be generated and stored in Middleware. 	|
+| `availableFrom`     | string  | **Optional** | yyyy-MM-dd					| thgpackageavailablefrom 				|
+| `availableTo`       | string  | **Optional** | yyyy-MM-dd					| thgexpiredate						|
+| `calculatedCount`   | integer | **Optional** | 						| thgcalculatedcount 					|
+| `consistsOfCount`   | integer | **Optional** | 						| thgconsistsofcount 					|
+| `consistsOfUnit`    | string  | **Optional** | From reference data, eg. "STK"			| thgconsistsofunit (rdunit) 				|
+| `dPakLayerCount`    | integer | **Optional** | Only "T-PAK"					| thgdpaklayercount 					|
+| `deliverable`       | boolean | **Optional** |						| thgcanbeordered 					|
+| `gtin`              | string  | **Optional** |						| thggtin 						|	
+| `height`            | decimal | **Optional** | millimeters					| thgheight 						|	
+| `length`            | decimal | **Optional** | millimeters					| thglength 						|
+| `maxStackingWeight` | decimal | **Optional** | Only "T-PAK"					| thgmaxstackingweight 					|
+| `minOrderQuantity`  | integer | **Optional** | Only "F-PAK"					| thgminorderquantity 					|	
+| `packageNumber`     | integer | **Optional** |						| __thgpackageid__ ?					|
```

Comment:
`packageNumber` is the sequential numbering for item packages which we are using today. So typically this would be 1 for F-PAK, 2 for D-PAK and so on. We have discussed this in the past, but did we come to a conclusion of who will generate these numbers?

Reply:
Ah yes thats correct. We need to change to TBD then.